### PR TITLE
First week at outreachy blogs

### PR DESCRIPTION
**Pull request by AryanGodara**
@pitag-ha 
I am an intern at Outreachy Summer'23. This blog describes my first week working at OCaml for the Outreachy internship. And I would like to link this blog as a post to Ocaml.org blog

### CODE REVIEW

**Comment by Burnleydev1** [comment]
@AryanGodara , I made an error with mine, I will just correct that now 😄

File: data/planet/outreachy/first-week-at-outreachy.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: First week experience at Outreachy
+description: To write down the experience of working at OCaml for outreachy internship
+url: https://medium.com/@aryangodara_19887/first-week-at-outreachy-54f4af71820f
+date: Tue Jun 13 16:36:50 IST 2023
+preview_image: https://miro.medium.com/v2/resize:fit:1400/format:webp/1*fLJmaO2fsscyePbR3Kgfkw.jpeg
+---
```
Comment (cuihtlauac) [inline data/planet/outreachy/first-week-at-outreachy.md]: You have to have some content there, it can't be empty

**Review by cuihtlauac — DISMISSED** [review]

File: data/planet/outreachy/first-week-at-outreachy.md (hunk 1 above)
Comment (AryanGodara) [reply data/planet/outreachy/first-week-at-outreachy.md]: I'm sorry I don't understand, Is the issue with the link to the image?

File: data/planet/outreachy/first-week-at-outreachy.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+title: First week experience at Outreachy
+description: To write down the experience of working at OCaml for outreachy internship
```
Comment (sabine) [inline data/planet/outreachy/first-week-at-outreachy.md]:
```suggestion
description: I share a bit about my experience of working at OCaml during my Outreachy internship.
```

**Pull request merged**